tes-adapter/src/main/java/com/epam/pipeline/tesadapter/service/TaskMapper.java

```diff
@@ -69,4 +77,75 @@ private TesExecutor getExecutorFromTesExecutorsList(List<TesExecutor> tesExecuto
                 "executors"));
         return tesExecutors.get(FIRST);
     }
+
+    public TesTask mapToTesTask(PipelineRun run) {
+        final TesTask tesTask = new TesTask();
+        tesTask.setId(String.valueOf(run.getId()));
+        tesTask.setState(createTesState(run.getStatus()));
+        tesTask.setName(run.getPodId());
+        tesTask.setResources(createTesResources(run));
+        tesTask.setExecutors(createListExecutor(run));
+        tesTask.setInputs(createTesInput(ListUtils.emptyIfNull(run.getPipelineRunParameters())));
+        tesTask.setOutputs(createTesOutput(ListUtils.emptyIfNull(run.getPipelineRunParameters())));
+        tesTask.setCreationTime(run.getStartDate().toString());
+        return tesTask;
+    }
+
+    private TesState createTesState(TaskStatus status){
+        TesState tesStatus = TesState.UNKNOWN;
```

Comment:
we can write this method without local variable, just use return statement in each case: 
```
case RUNNING:
    return TesState.RUNNING;
......
case STOPPED:
    return TesState.CANCELED;
default:
    TesState.UNKNOWN;
```